src/ocl/convolutionocl.cpp

```diff
@@ -827,16 +827,21 @@ void ConvolutionDescriptor::GetSolutionsFallback(Handle& handle,
     ctx.SetStream(&handle);
     ctx.DetectRocm();
 
+    const auto wti2time = [](const float& wti) {
```

Comment:
Please write your own conversion function.

Reply:
Added the function in my refactor for AI timing.